plugins/view/dummyView/view_template.html

```diff
@@ -0,0 +1,10 @@
+{# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved. #}
```

Comment:
I think I will need to create a `MANIFEST.in` file to include those non-Python files